### ISSUE TITLE
pip install misses some files

### ISSUE THREAD

**Issue by fekad** [hampusnasstrom/metainfo-yaml2py#16]
```
FileNotFoundError: [Errno 2] No such file or directory: '.venv/lib/python3.9/site-packages/metainfoyaml2py/resources/standard_file_content.yaml'
```

Suggestion: https://setuptools.pypa.io/en/latest/userguide/datafiles.html

**Comment by hampusnasstrom** [comment]
Oh, right. Thanks @fekad! I forgot to add the resources when I moved from setup.cfg to pyproject.toml...

**Comment by hampusnasstrom** [comment]
> Oh, right. Thanks @fekad! I forgot to add the resources when I moved from setup.cfg to pyproject.toml...

I guess, the resource should now be placed inside the package.

**Comment by hampusnasstrom** [comment]
@fekad this should now be fixed from version 0.0.2. Thanks for the help and please let me know if you run into further issues.
